index.html

```diff
@@ -0,0 +1,147 @@
+                    </li>
+                    <li>
+                        <a href="#join">Join</a>
+                    </li>
+                    <li>
+                        <a href="#partners">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+
+    <!-- MAIN SECTION STARTING -->
+    <main>
+        <section class="main">
+            <nav class="Mobile-navbar">
+                <img id="cancelBtn" src="images/Icon - Cancel.svg" alt="">
+                <i id="hamburger" class="fas fa-bars hamburger" aria-hidden="true"></i>
+                <ul id="hamburger-list">
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="">Join</a>
+                    </li>
+                    <li>
+                        <a href="">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="container">
+                <div class="content">
+                    <p class="content-hellow">"Hellow! Music World"</p>
+                    <h1 class="main-content-heading">ZIRO MUSIC FESTIVAL NIGHT 2023</h1>
+                    <p class="content-detail">A Ziro Music Festival is a live music performance in front of an audience. 
+                        The performance may be by a single musician, sometimes then called a recital, 
+                        or by a musical ensemble, such as an orchestra, choir, or band. 
+                    </p>
+                    <h2 class="content-secondHeading">2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p class="content-lastline">@ National Music of universe.</p>
+                </div>
+        </div>
+        </section>
+
+        <!-- ------------PROGRAM SECTION STARTING------------- -->
+        <Section class="program" id="join">
+            <h3>Main Program</h3>
+            <div id="allcardS" class="cards"> </div>
+            <button type="button">Join Our Concert</button>
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </Section>
+
+        <!-- -----------------Guest section starting--------------- -->
+        <section class="guests">
+            <h2>Guests</h2>
+            <article id="guest-list" class="guest-list"> </article>
+            <button  id="seeMore" type="button">More <i class="fas fa-caret-down" aria-hidden="true"></i> </button>
+        </section>
```

Comment:
This section of your app is not matching the provided design.  Applies to both desktop and mobile version.

- [ ] The size of your images is proportionately large compared to provided design and doesn't look good. Consider decreasing the size of your images.
- [ ] The font size, style, and color you are using for the subheading are not correct. The design color is `#EC5242` and style is `italic` and font-size is also smaller. However, in your design, all these are miss matches. Kindly consider fixing these.
- [ ] The images use certain effects using the `before` property which is missing in your case.
- [ ] Certain texts are underlined in the provided design, however, in your design, they are not. 

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/bfd3f4ba-3b73-4779-8c27-39adf85ce760) | ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/9f919572-accf-4e21-904c-8d70e53a7551) |

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/ec6883b4-3669-4d23-a3fc-e6f86e9e6dc7) | ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/738b3087-4685-458f-b634-e0839bc82118) |

Reply:
Fixed.